src/frontend/efiling-frontend/README.md

```diff
@@ -4,31 +4,47 @@ Welcome to the eFiling frontend!
 
 ## Running the Frontend
 
+In the project directory `efiling-frontend`:
+
 Set the required environment variables locally. Create a `.env.development` file and populate the environment variables as shown in the `.env.example` file.
 
+In the root directory `jag-file-submission`, run:
+
+```bash
+docker-compose up -d efiling-api efiling-demo keycloak keycloak-config redis redis-commander clamav
+```
+
 In the project directory (`efiling-frontend`), you can run:
 
-### `yarn install`
+```bash
+yarn install
+```
 
 Installs all the required dependencies to get the application up and running.
 
-### `yarn start`
+```bash
+yarn start
+```
 
 Runs the app in the development mode.<br />
```

Comment:
```suggestion
Runs the app in the development mode.

```

Reply:
Sounds good. I only changed the yarn start lines, but these additional changes would also improve the readme.  Thanks!